### ISSUE TITLE
New "Error", Steam 3 hour downtime might be patch to block bots?

### ISSUE THREAD

**Issue by mtzi** [Maxx53/SCMBot#487]
Title sais it
![bot](https://cloud.githubusercontent.com/assets/12753786/10579132/161f8f9a-767f-11e5-9696-65788deed9b4.PNG)

So, WHAT IS THIS?

**Comment by mtzi** [comment]
Going to be off for the next 4 hours, i'll be back if someone has anything similar etc to talk.

**Comment by X4ndrius** [comment]
@mtzi Time lock with a large number of requests has increased by 5 minutes to several hours. The temporary measure or not is unknown. It is necessary to test after the start of server stability.

**Comment by mazzolank** [comment]
banned ip

**Comment by KeooR** [comment]
Maybe for increase price of items on steam.

**Comment by mtzi** [comment]
Still have it, its been about 3 hours. I've never seen this before, so i dont know what to think.

@mazzolank i doubt its IP, if they ban you, they tell you they banned you. It sais "later"

@X4ndrius Need to test how many hours... Its 4 hours 15 mins as of now.

**Comment by mtzi** [comment]
@mazzolank, It was TRULY IP ban. I dont pay for "Static IP" so i have dynamic IP, so if i restart router i get new IP every time. Restarted, error fixed. Thanks!

**Comment by X4ndrius** [comment]
@mtzi Yes reboot router fix it. Increase Delay between requests to >500. And Simultanius request to 1

**Comment by Alex1os** [comment]
It's a 6 hour temporary market ban

Lg Alex

**Comment by KeooR** [comment]
Somebody have find a new stable delay for don't be ban ? 
500 delay and 0 delay between requests give me a ban.

**Comment by X4ndrius** [comment]
@KeooR Increases delay. Or change ip every lock.

**Comment by KeooR** [comment]
@X4ndrius 500 delay and 500 delay between requests give me also a ban, did you find stable delay ?

**Comment by mtzi** [comment]
@KeooR I have worked for some hours, Delay between requests 1000, item delay = Number of items * 1000
Example: i have 6 items, so Item delay for every item = 6000, if you have 4 items item delay 4000 etc
no ban for 3-4 hours i think. also bought 1 item

**Comment by X4ndrius** [comment]
@KeooR Stable delay has not been found.

**Comment by KeooR** [comment]
@mtzi I have 1 item, so I'm going to try Delay between requests 1000 and 100 delay item.

**Comment by X4ndrius** [comment]
@KeooR With these settings, a few hours before the scan ban.

**Comment by mtzi** [comment]
@KeooR @X4ndrius  @Maxx53 
I sat down and made some "tests", slow scans, etc... here is results.
Bot died in 1 hour 42 min, or 102min of NONSTOP requests (median 1.2 per sec, 8 per 10 sec)
As shown below, bot died at approx. 4931 requests, of course its not exact could be anything from 4800 to 5000.
My idea behind this is that they block u at 100 mins of nonstop requests, OR at 5000 requests within some time limit, maybe the whole day? 
![bot](https://cloud.githubusercontent.com/assets/12753786/10609944/8801eec6-774e-11e5-920c-1188fe8d14d0.PNG)

**Comment by X4ndrius** [comment]
@mtzi 100 min scan. My tests show similar results. It remains to find a cooldown when you stop scanning.

**Comment by KeooR** [comment]
So we are limited to 5000 request per day you think ?

**Comment by X4ndrius** [comment]
@KeooR I think the 5000 queries per 100 minutes.

**Comment by mtzi** [comment]
It is either 5000 requests per "SOME HOURS" the exact number is unknown, or at 100 minutes if we send requests all time, no-stop we get ban, 1 of the 2, not both. i will start test again in some hours with slower scan and see exactly how it works. its 1 of the 2, 5000 requests or 100 min non-stop
1 of the 2 things causes ban

**Comment by mtzi** [comment]
in 2:30 hours from now i'll know what bans, minutes or requests. Will report back with results.

**Comment by KeooR** [comment]
Ok @mtzi thank, and you and use 1500 delay per item ?

**Comment by mtzi** [comment]
the thought is:  Item delay > number of items * Delay between requests
![aaaaaa](https://cloud.githubusercontent.com/assets/12753786/10613592/7e74a230-775e-11e5-82de-f9d75de39ce5.PNG)

This is screenshot. This is TEST. maybe is ban

**Comment by mtzi** [comment]
2:30 hours -> 150 mins
5400 requests, still no ban.

going strong, will report results back in a few hours again

**Comment by mazzolank** [comment]
how many good setting delay ,no ban

**Comment by KeooR** [comment]
@mtzi Thanks i love u mtzi for do that :)

**Comment by X4ndrius** [comment]
@mtzi Wery THX bro

**Comment by mtzi** [comment]
Still going, 3 hours and 54 mins later. @KeooR @X4ndrius 
@mazzolank picture of settings 2 posts above you. 6 item, 1500 and 10000. look picture.

**Comment by X4ndrius** [comment]
@mtzi Thx man. I wonder how is purchased with such settings? Sorry for bad English.

**Comment by mtzi** [comment]
5 hours and going.

@X4ndrius I don't know.. i got 2 items, but more expensive items, that are slow (M4A4 Dragon King Field-Tested, M4A4 Evil Daimyo), 2 buy in 5 hours

**Comment by mtzi** [comment]
5 hour 5 minute IP BAN!

**Comment by Rocky54** [comment]
If I use 1 item, what settings do I put in?


P.S. https://translate.google.ru/

**Comment by KeooR** [comment]
@mtzi see that 
![noban](https://cloud.githubusercontent.com/assets/13817999/10622192/0ee7607e-7786-11e5-9035-2f7849a498c6.PNG)
I use that settings no-stop and no ban...

**Comment by KeooR** [comment]
@mtzi see that
![noban](https://cloud.githubusercontent.com/assets/13817999/10622233/59d72542-7786-11e5-871b-a98de90a70fb.PNG)
I use that settings no-stop and i have no ban...

**Comment by X4ndrius** [comment]
@KeooR it is only two hours had passed.We need to test more

**Comment by mazzolank** [comment]
Help me , new device restar ip
![untitled](https://cloud.githubusercontent.com/assets/14069895/10627598/24723af0-77ee-11e5-9b44-a54ac993df87.jpg)

**Comment by X4ndrius** [comment]
@mazzolank Standard restriction on steam for the new device. Wait 7 days.

**Comment by KeooR** [comment]
@mazzolank You have reset your Browser ?

**Comment by mazzolank** [comment]
@KeooR 
how to change the ip without resetting the modem

**Comment by KeooR** [comment]
@mazzolank go to your modem settings ( paste this in your browser : 192.168.1.1 ) Disconnect your modem form the internet, after reconnect.
